libass/ass.h

```diff
@@ -734,6 +734,24 @@ void ass_process_chunk(ASS_Track *track, const char *data, int size,
  */
 void ass_set_check_readorder(ASS_Track *track, int check_readorder);
 
+/**
+ * \brief Garbage collect events that preceed deadline
+ * \param track track
+ * \param deadline cut-off timestamp in milliseconds.
+*/
+void ass_prune_events(ASS_Track *track, long long deadline);
+
+/**
+ * \brief Configure the automatic garbage collection of events.
+ * \param track track
+ * \param enabled enable automatic garbage collection with the given delay.
+ * \param delay delay from "now" (ass_render_frame) in milliseconds.
+ * After every succesful render, events whose undisplay timestamp predate
+ * "now - delay" will be deleted. Delay of 0 performs aggressive collection.
+ * Disabled by default (no collection, keep all parsed events).
+ */
+void ass_configure_gc(ASS_Track *track, int enabled, long long delay);
```

Comment:
There’s currently no way to disable automatic gc again without recreating the track. negative delay/backbuffer values can be used and documented for this

Reply:
Isn't `gc_enabled`, set by `int enabled` of `ass_configure_gc` letting the user change the behaviour on the fly, or am I missing something?

Reply:
nope you’re not; *I* was missing/forgetting `enabled`

Most of our other API doesn’t take a dedicated `enabled` using zero and/or negative values instead, but having a clear, explicit toggle isn’ŧ a bad thing